pandas/core/frame.py

```diff
@@ -2504,9 +2504,14 @@ def transpose(self, *args, **kwargs):
 
         Parameters
         ----------
-        *args, **kwargs
-            Additional arguments and keywords have no effect but might be
-            accepted for compatibility with numpy.
+        *args : tuple, optional
```

Comment:
don't we *still* need kwargs for this?

Reply:
AFAICT, no. Neither `np.transpose` nor `ndarray.transpose` take additional keyword arguments.